php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -630,14 +637,7 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
                     break;
                 case 'numeric':
                     if ($addElements) {
-                        $options = array();
-                        if (!empty($pieces[3])) {
-                            $options['min'] = $pieces[3];
-                        }
-                        if (!empty($pieces[4])) {
-                            $options['max'] = $pieces[4];
-                        }
-                        $this->addNumericElement($pieces[1], $pieces[2], $options);
+                        $this->addNumericElement($pieces[1], $pieces[2]);
```

Comment:
Is this change Ok? No more `min` nor `max` in the element.

Reply:
@johnsaigle still not resolved. Please comment if this is the intent and why as it modify the functionality

Reply:
@PapillonMcGill As I commented above, the `$options` variable is never used in the `addNumericElements` function so I removed it from the function. I also modified the calling code here because we don't need to create an array that is never used.

Reply:
@johnsaigle Right, must still be under influence from the parties. Hadn't see that min and max are in $option.